### PR TITLE
Add feature

### PR DESCRIPTION
**Pull request by Sathiyakugan**

### CODE REVIEW

File: features/org.wso2.carbon.identity.developer.language-server.feature/pom.xml [hunk 1]
```diff
@@ -0,0 +1,141 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (darshanasbg) [inline features/org.wso2.carbon.identity.developer.language-server.feature/pom.xml:1]: Add license

File: features/etc/feature.properties [hunk 2]
```diff
@@ -0,0 +1,200 @@
+#
+# Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (gayashanbc) [inline features/etc/feature.properties:2]: Is the year correct?

File: features/org.wso2.carbon.identity.developer.language-server.feature/pom.xml [hunk 3]
```diff
@@ -0,0 +1,158 @@
+<!--
+  ~ Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (gayashanbc) [inline features/org.wso2.carbon.identity.developer.language-server.feature/pom.xml:2]: Is the year correct?

**Review by gayashanbc — COMMENTED** [review]
Check the year in the license headers

File: pom.xml [hunk 4]
```diff
@@ -79,7 +80,43 @@
             </releases>
         </repository>
     </repositories>
-  
+
```
Comment (NipuniBhagya) [inline pom.xml:83]: Let's remove this nnecessary new line

File: features/org.wso2.carbon.identity.developer.language-server.feature/pom.xml (hunk 1 above)
Comment (Sathiyakugan) [reply features/org.wso2.carbon.identity.developer.language-server.feature/pom.xml]: Fixed with https://github.com/wso2-extensions/identity-tools-debugger/pull/17

File: features/etc/feature.properties (hunk 2 above)
Comment (Sathiyakugan) [reply features/etc/feature.properties:2]: Fixed with #17

File: features/org.wso2.carbon.identity.developer.language-server.feature/pom.xml (hunk 3 above)
Comment (Sathiyakugan) [reply features/org.wso2.carbon.identity.developer.language-server.feature/pom.xml:2]: Fixed with #17

File: features/etc/feature.properties (hunk 2 above)
Comment (Sathiyakugan) [reply features/etc/feature.properties:2]: Fixed with #17

File: pom.xml (hunk 4 above)
Comment (Sathiyakugan) [reply pom.xml:83]: Fixed with #17

File: features/org.wso2.carbon.identity.developer.language-server.feature/pom.xml (hunk 1 above)
Comment (Sathiyakugan) [reply features/org.wso2.carbon.identity.developer.language-server.feature/pom.xml]: Fixed with #17

**Pull request merged**